packages/next/client/image.tsx

```diff
@@ -510,6 +510,17 @@ export default function Image({
           ) : null}
         </div>
       ) : null}
+      <noscript>
+        <img
+          {...rest}
+          {...imgAttributes}
+          src={src}
+          decoding="async"
+          sizes={sizes}
+          style={{ ...imgStyle, visibility: 'visible' }}
```

Comment:
This should be `inherit`, not `visible`.

Reply:
good point